tox.ini

```diff
@@ -51,16 +51,16 @@ deps =
     flake8-docstrings-complete>=1.0.3
     flake8-test-docs>=1.0
 commands =
-    pydocstyle {[vars]src_path}
+    pydocstyle {[vars]all_path}
     codespell {toxinidir} --skip {toxinidir}/.git --skip {toxinidir}/.tox \
```

Comment:
I remove codespell from my CI, feels very unnecessary.

Reply:
Codespell is one of the tools necessary for the [operator workflows](https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#static-code-analysis) to function properly.